From 111d6030f04c17ad6af488ae9b041b6a3a2d38e1 Mon Sep 17 00:00:00 2001
From: Eugeniu Rosca <erosca@de.adit-jv.com>
Date: Fri, 28 Sep 2018 11:27:49 +0200
Subject: [PATCH 152/909] soc: renesas: rcar-avs: Fix using plain integer as
 NULL warning

commit 8a06eca5c7461aadebc539636c3b781acc6a0a19 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Silent sparse (v0.5.2-1-ga3c4716a703a) complaint:
drivers/soc/renesas/rcar-avs.c:36:75: warning: Using plain integer as NULL pointer

Suggested-by: George G. Davis <george_davis@mentor.com>
Signed-off-by: Eugeniu Rosca <erosca@de.adit-jv.com>
Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar-avs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/renesas/rcar-avs.c b/drivers/soc/renesas/rcar-avs.c
index 2d1aade78872..7c19f74309cc 100644
--- a/drivers/soc/renesas/rcar-avs.c
+++ b/drivers/soc/renesas/rcar-avs.c
@@ -33,7 +33,7 @@ static int change_default_opp_pattern(struct device_node *avs_node,
 	int size;
 	struct property *pp;
 
-	dev_nums = of_count_phandle_with_args(avs_node, "target_devices", 0);
+	dev_nums = of_count_phandle_with_args(avs_node, "target_devices", NULL);
 
 	for (i = 0; i < dev_nums; i++) {
 		dev_node = of_parse_phandle(avs_node, "target_devices", i);
-- 
2.17.1

